src/components/shared/frame/Header/index.tsx

```diff
@@ -88,22 +86,6 @@ export const Header = ({
     )
   }, [companyData, companyDetails])
 
-  useEffect(() => {
-    if (
-      companyDetails?.companyRole.includes(
-        CompanyRoleEnum.ONBOARDING_SERVICE_PROVIDER
```

Comment:
Please also remove ONBOARDING_SERVICE_PROVIDER from CompanyRoleEnum  from userApiSlice.ts

Reply:
why? This is being used in ConnectorTypeSelection.tsx

Reply:
Please use COMPANY_ROLES enum from constants.ts file and remove CompanyRoleEnum(from userApiSlice.ts) and its usage as it is a duplicate enum

Reply:
done